src/screens/ContactScreen/index.js

```diff
@@ -23,18 +13,17 @@ import {COLORS} from '../../utils/theme';
 import {Context} from '../../context';
 import {Loading} from '../../components';
 import {Search} from './elements';
-import {setChannel} from '../../context/actions/setChannel';
 import {userPopulate} from '../../service/users';
 import {withInteractionsManaged} from '../../components/WithInteractionManaged';
+import useCreateChat from '../../hooks/screen/useCreateChat';
+import {DEFAULT_PROFILE_PIC_PATH} from '../../utils/constants';
 
 const {width} = Dimensions.get('screen');
 
 const ContactScreen = ({navigation}) => {
   const [loading, setLoading] = React.useState(false);
   const [users, setUsers] = React.useState([]);
   const [profile] = React.useContext(Context).profile;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of `setChannel` and `client` from the context might cause issues if they are used elsewhere in the component. Please ensure that these variables are not required later in the code or their functionality is replaced appropriately.



<!-- This is an auto-generated comment by OSS CodeRabbit -->